resource-managers/mesos/src/main/scala/org/apache/spark/deploy/mesos/config.scala

```diff
@@ -58,13 +58,40 @@ package object config {
 
   private[spark] val DRIVER_LABELS =
     ConfigBuilder("spark.mesos.driver.labels")
-      .doc("Mesos labels to add to the driver.  Labels are free-form key-value pairs.  Key-value " +
+      .doc("Mesos labels to add to the driver.  Labels are free-form key-value pairs. Key-value " +
         "pairs should be separated by a colon, and commas used to list more than one." +
         "Ex. key:value,key2:value2")
       .stringConf
       .createOptional
 
-  private[spark] val DRIVER_FAILOVER_TIMEOUT =
+  private[spark] val SECRET_NAME =
+    ConfigBuilder("spark.mesos.driver.secret.name")
+      .doc("A comma-separated list of secret references. Consult the Mesos Secret protobuf for " +
```

Comment:
If this is a list it should be created with `.toSequence`; it returns the value properly parsed as a list to you. And it should probably be called `.names`.